docs/adrs/0550-Credscan-for-secret-scan.md

```diff
@@ -0,0 +1,20 @@
+# ADR 0550: Use CredScan for credential scanning
+
+**Date**: 2020-06-22
+
+**Status**: Proposed
+
+## Context
```

Comment:
any issue with this feature I can check? want to make sure the product is onboard with this.
cc. @chrispat